jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -373,3 +383,129 @@ def vue_do_aper_phot(self, *args, **kwargs):
             self.radial_plot = self._fig
             self.bqplot_figs_resize = [self._fig]
             self.plot_available = True
+
+
+# NOTE: These are hidden because the APIs are for internal use only
+# but we need them as a separate functions for unit testing.
+
+def _radial_profile(radial_cutout, reg_bb, aperture, raw=False):
+    """Calculate radial profile.
+

```

Comment:
should the license note be here as well (just in case this function is ever moved around)?

Reply:
Good catch! Moved.